website/docs/r/kms_key_policy.html.markdown

```diff
@@ -0,0 +1,89 @@
+---
+subcategory: "Key Management Service"
+layout: "ibm"
+page_title: "IBM : kms-key-policies"
+description: |-
+  Manages key policies for IBM hs-crypto and kms.
+---
+
+# ibm\_kms_key_policies
+
+Provides a resource to manage key policies for hs-crypto and key-protect services. This allows key policies to be created. Key policies can be created for an existing kms key resource.
+
+
+## Example usage to create a Key and associate a key policy.
+
+```terraform
+resource "ibm_resource_instance" "kms_instance" {
+  name     = "instance-name"
+  service  = "kms"
+  plan     = "tiered-pricing"
+  location = "us-south"
+}
+
+resource "ibm_kms_key" "key" {
+  instance_id = ibm_resource_instance.kp_instance.guid
+  key_name       = "key"
+  standard_key   = false
+}
+
+resource "ibm_kms_key_policies" "keyPolicy" {
+  instance_id = ibm_resource_instance.kms_instance.guid
+  key_id = ibm_kms_key.key.key_id
+     rotation {
```

Comment:
remove additional tab before rotation..

run terraform fmt on this example and then add that template here